vim/vimrc.symlink

```diff
@@ -11,21 +11,17 @@ Plug 'tpope/vim-fugitive' " amazing git wrapper for vim
 Plug 'tpope/vim-surround' " mappings to manage surroundings in pairs, such as quotes, parens..
 Plug 'tpope/vim-endwise' " automatically add end in ruby
 Plug 'mileszs/ack.vim' " search inside files using ack
-Plug 'benekastah/neomake' " neovim replacement for syntastic using neovim's job control functonality
+Plug 'w0rp/ale' " asynchronous lint engine
```

Comment:
Ale is more specific, it's made only for linters. There is support for bunch lint tools out of the box, it can also detect local version of linter (from `node_modules` or `vendor/bundle`). At the end of the day it's easier to setup and it actually worked as expected in contrast to neomake which didn't.

Lately i like more and more plugins that support multiple languages out of box instead of one plugin per language, for example [vim-polyglot](https://github.com/sheerun/vim-polyglot) and [vim-snipmate](https://github.com/garbas/vim-snipmate)